### ISSUE TITLE
DatabaseException(file/data is not in the requested format, code: wrongFormat)

### ISSUE THREAD

**Issue by natgross** [cbl-dart/cbl-dart#337]
I suddenly get this exception when trying to create a new db:
`DatabaseException(file/data is not in the requested format, code: wrongFormat)`

This, with the beta packages, as well as with the latest 1.x versions.

**I don't think this is a bug from you part.** 
I recently changed the home folder for the db files due to Android restrictions not allowing the use of public external directories.
However, am not sure if this is related, since I have had no problems with this in the past.

Any insight you can offer, please?

**Comment by blaugold** [comment]
A Couchbase Lite database is a directory with files in it. You typically get this error when the directory exists, but the contents are not as Couchbase Lite expects them to be.

I could see this error occurring when asynchronously opening a new database twice without wait for one of the calls to `Database.openAsync` to finish before starting the other one.

**Comment by natgross** [comment]
> I could see this error occurring when asynchronously opening a new database twice without waiting for one of the calls to Database.openAsync to finish before starting the other one

Correct. It seems like I'm doing something along those lines. Need to check some more. 
I'm closing this out, and if I have anything to add to this thread, I will do so.
Thanks
